platform/backend_connectors/hydrolix_backend_connector.go

```diff
@@ -4,16 +4,23 @@
 package backend_connectors
 
 import (
+	"bytes"
 	"context"
 	"database/sql"
+	"fmt"
 	"github.com/QuesmaOrg/quesma/platform/config"
+	"github.com/QuesmaOrg/quesma/platform/logger"
+	"io"
+	"net/http"
 
 	quesma_api "github.com/QuesmaOrg/quesma/platform/v2/core"
 )
 
 type HydrolixBackendConnector struct {
 	BasicSqlBackendConnector
-	cfg *config.RelationalDbConfiguration
+	cfg         *config.RelationalDbConfiguration
```

Comment:
I think some variation of the comment you made in Exec function: 
```// TODO for fallback, execute the query directly on the database connection```
would be useful here in this structure definition as well.